.travis.yml

```diff
@@ -31,58 +25,55 @@ services:
 #  build matrix, i.e. every build configuration.
 #  In turn, variables defined in the 'matrix' section each triggers a separate
 #  build.
-#
-#  Please note that test fails on first error as default
-#  To skip errors and fail silently set environment variable:
-#    SKIP_TEST_ON_ERROR=1
-#
-#  To speed up installation of brew formulas on osx auto update for homebrew is
-#  turned off by setting HOMEBREW_NO_AUTO_UPDATE=1
 env:
   global:
-    - MYSQL_SERVER=127.0.0.1
+    - COMPILER_VERSION=
+    - DEPLOY_PACKAGE=false
+    - DIST_NAME=ubuntu
+    - DIST_VERSION=xenial
```

Comment:
I'd mention that this commit also stops building for trusty and starts building for xenial.